### ISSUE TITLE
[A11y][Contrast]Color contrast for the focus on icons Facebook and Twitter with background is 1.658 which is less than 3:1

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8185]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184494
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Hit the URL and login with appropriate credentials
    Tab till "Packages tab" item and press enter
    Tab till any package and press enter.
    Tab till "Share" section
    Check whether the color contrast for the background to focus on the icons "Facebook" and "Twitter" greater than or equal to 3:1

## Actual Results:

Color contrast for the focus on icons Facebook and Twitter with background is 1.658 which is less than 3:1

   
## Expected Results:
Color contrast for the focus on icons Facebook and Twitter with background is 1.658 it should be greater than or equal to 3:1
## User Impact:
Users with low vision would face difficulty to identify the elements if the non-text contrast is less than 3:1

## Notes
We don't control these icons. I don't know if we are allowed to/should change them.
